### ISSUE TITLE
Typo in Matlab cp_mass documentation

### ISSUE THREAD

**Issue by bryanwweber** [Cantera/cantera#590]
### Cantera version
2.4.0

The documentation for the `cp_mass` function in the Matlab toolbox is incorrect, it reads

```matlab
% :return:
%     Molar basis specific heat of the mixture at
```

To correct this, the file at [`interfaces/matlab/toolbox/@ThermoPhase/cp_mass.m`](https://github.com/Cantera/cantera/blob/master/interfaces/matlab/toolbox/%40ThermoPhase/cp_mass.m) needs to be edited, and the other similar files that distinguish mass- or molar-basis should be checked for consistency.

As reported by email by Qiang Zhange.

**Comment by arjavvashi** [comment]
I want to work on this issue. Is it still open? I can see @CyberDrudge's pull request.

**Comment by bryanwweber** [comment]
@arjavvashi This issue will be closed by #591. No need to work on it.
